Archive/PiBase/Properties.lean

```diff
@@ -0,0 +1,21 @@
+import Mathlib.Topology.Basic
+import Mathlib.Topology.Separation
+import Mathlib.Order.Filter.Basic
+
+open Filter Topology
+
+universe u
+variable (X : Type u) [TopologicalSpace X]
+
+namespace πBase
+
+def P1 := T0Space X
```

Comment:
Making these ‘abbrev’ instead of ‘def’ should probably make it easier to work with them

Reply:
In this case, I suppose there would be to be some logic to handle when properties are defined in mathlib directly and when they aren't. Or we could make every PN an `abbrev` and then also provide separate `def`s for those who need them, e.g. `abbrev P99 := UsSpace` and also `def UsSpace [...]` since US spaces aren't in mathlib proper.

Reply:
I think for rewrites, this might be the case. It should help for type class inference!